### PR TITLE
Added Twilio Auth Service

### PR DESCRIPTION
**Pull request by sahithm06**

### CODE REVIEW

File: BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/controller/AuthController.java [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
-import org.springframework.web.bind.annotation.RequestMapping;
 
-public class AuthController {
+import com.techhunters.borrowmyproducts.service.TwilioAuthService;
+
+@Controller
+public class AuthController
+{
+	@Autowired
+	TwilioAuthService twilioAuthService;
```
Comment (ghost) [inline BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/controller/AuthController.java]: Use constructor injection with final variables

File: BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/service/AuthService.java [hunk 2]
```diff
@@ -1,4 +1,17 @@
 package com.techhunters.borrowmyproducts.service;
 
-public interface AuthService {
+
+public interface AuthService 
+{
+	//By Default the parameters are public static final 
+	 String ACCOUNT_SID = "ACa95f20b59d30fc54a834d259c85df35d";
```
Comment (ghost) [inline BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/service/AuthService.java]: never put keys and password in java code. Put them is application.properties and fetch from there

File: BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/service/TwilioAuthService.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        //just printing on the console for verification
+        //System.out.println(verification.getStatus().toUpperCase());
+        
+        return true;
+       }
+      
+       catch(Exception exception)
+       {
+    	  
+    	   return false;
```
Comment (ghost) [inline BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/service/TwilioAuthService.java]: print the exception for debugging

**Review by ghost — CHANGES_REQUESTED** [review]
Also indent your code properly

File: BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/service/TwilioAuthService.java (hunk 3 above)
Comment (sahithm06) [reply BorrowMyProducts/src/main/java/com/techhunters/borrowmyproducts/service/TwilioAuthService.java]: thank you

**Pull request closed without merging**